core/src/main/java/org/apache/iceberg/FastAppend.java

```diff
@@ -141,6 +144,18 @@ private ManifestFile copyManifest(ManifestFile manifest) {
     return newManifests;
   }
 
+  @Override
+  public Object updateEvent() {
+    long snapshotId = snapshotId();
+    long sequenceNumber = ops.refresh().snapshot(snapshotId).sequenceNumber();
```

Comment:
The commit is going to update the current metadata, so there shouldn't be a need to `refresh` here. You'll get the correct information using `current`.

Reply:
fixed